.chloggen/mongodb-update-supported-versions.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: receiver/mongodb
```

Comment:
I've seen other people specify the receiver block this way for changelogs but have also seen people specify `mongodbreceiver`. Should this be something we should match so the changelog is not so inconsistent or is it handled somewhere?

Reply:
Good catch. Fixed